components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,301 @@
+| Red Hat Enterprise   |  7, 8 (For 8 or above                         |
+| Linux                | versions, **SELinux** configuration must be   |
+|                      | permissive. By default, in RHEL               |
+| (64 Bit OS)          | 8, **SELinux** configuration is enforced).    |
+|                      |                                               |
+|                      |                                               |
+|                      |                                               |
+|                      | Red Hat Enterprise Linux derivatives include  |
+|                      | Amazon Linux v1 (using RHEL 6 packages) and   |
+|                      | v2 (using RHEL 7 packages).                   |
++----------------------+-----------------------------------------------+
+| Ubuntu (64 Bit OS)   |  14.04.x, 16.04.x, 18.04.x, 20.04.x           |
++----------------------+-----------------------------------------------+
+| Centos (64 Bit OS)   |  7                                            |
++----------------------+-----------------------------------------------+
+
+## Virtual Machine (VM) instances Type 
+
++--------------+-------------+--------------------------+--------------+
+|              |       Type  |                    RAM   |              |
+| Instance     |             |                          | Volume-size  |
++==============+=============+==========================+==============+
+| PostgreSQL   | t3a.medium  | 4GB RAM For Test and 8G  | 50GB         |
+|              |             | For Prod                 | (dedicated   |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
++--------------+-------------+--------------------------+--------------+
+| El           | m5a.large   | 8GB RAM For Test and 16G | 50G          |
+| asticsearch  |             | For Prod                 | B (dedicated |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
+|              |             |                          |              |
+|              |             |                          |              |
++--------------+-------------+--------------------------+--------------+
+| Automate     | t3a.medium  | 4GB RAM For Test and 8G  | 50G          |
+|              |             | For Prod                 | B (dedicated |
```

Comment:
same of this it should be t3.medium instead of t3a.medium

Reply:
done